packages/console/src/components/Dropdown/index.module.scss

```diff
@@ -0,0 +1,30 @@
+@use '@/scss/underscore' as _;
+
+.content {
+  background: var(--color-on-primary);
+  box-shadow: var(--shadow-light-s2);
+  border-radius: _.unit(2);
+  position: absolute;
+  min-width: 180px;
+
+  &:focus-visible {
+    outline-color: transparent;
```

Comment:
Shouldn't we set the focus-visible outline color to something like `--color-focus-variant`?

Reply:
Need to remove focus effect here, changed into `outline: none`.